src/pages/Transfer/TransferForm/index.tsx

```diff
@@ -0,0 +1,220 @@
+
+import * as React from 'react';
+import {
+  useSelector,
+  useDispatch
+} from 'react-redux';
+import { useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import clsx from 'clsx';
+import { toast } from 'react-toastify';
+import { newMonetaryAmount, CurrencyUnit } from '@interlay/interbtc-api';
+import { Currency } from '@interlay/monetary-js';
+
+import TokenAmountField from '../TokenAmountField';
+import {
+  Tokens,
+  TokenOption
+} from 'components/Tokens';
+import SubmitButton from 'components/SubmitButton';
+import FormTitle from 'components/FormTitle';
+import TextField from 'components/TextField';
+import ErrorModal from 'components/ErrorModal';
+import {
+  ParachainStatus,
+  StoreType
+} from 'common/types/util.types';
+import { showAccountModalAction } from 'common/actions/general.actions';
+import isValidPolkadotAddress from 'utils/validation/validate-address';
+import STATUSES from 'utils/constants/statuses';
+import {
+  KUSAMA,
+  POLKADOT
+} from 'utils/constants/relay-chain-names';
+
+const TRANSFER_AMOUNT = 'wrapped-token-input-amount';
+const RECIPIENT_ADDRESS = 'collateral-token-address';
```

Comment:
@tomjeatt 
It's a minor thing and it does not make any difference. But can we harmonize this kind of thing across the app?
e.g.
```
const TRANSFER_AMOUNT = 'transfer-amount';
```